modules/my_health/spec/serializer/v1/prescription_details_serializer_spec.rb

```diff
@@ -241,6 +241,6 @@
     expect(tracking_list).to be_an(Array)
     tracking_item = tracking_list[0]
     expect(tracking_item).to be_a(Hash)
-    expect(tracking_list[0]).to include({ 'carrier' => 'UPS', 'complete_date_time' => '2023-03-28T04:39:11-04:00' })
+    expect(tracking_list[0]).to include({ 'carrier' => 'UPS', 'completeDateTime' => '2023-03-28T04:39:11-04:00' })
```

Comment:
Can you explain what caused this to change? @stevenjcumming

Reply:
Active Model Serializer gem was snake casing the keys. `jsonapi-serializer` doesn't do that. The origin of the tracking list is in this factory:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/factories/prescription_details.rb#L101

Reply:
Ah, thanks. I was thinking why would only this statement change, but I also didn't realize we we're mixing snake case and camel case in that object. Not important now, but we'll probably want to change that in the future.